src/index.ts

```diff
@@ -5,6 +5,7 @@ import { logRequestErrors } from 'probot/lib/middleware/log-request-errors'
 
 import { opencollective } from './bot'
 import { validator } from './validate'
+import dotenv from 'dotenv'
```

Comment:
Is that enough to load the `.env` file? Is it working for you?

Reply:
No that's just an import.  

Seems I'll write the full codes to load it within the readme